packages/react/src/components/FluidNumberInput/FluidNumberInput.stories.js

```diff
@@ -0,0 +1,147 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import { FluidNumberInput, FluidNumberInputSkeleton } from '.';
+import {
+  ToggletipLabel,
+  Toggletip,
+  ToggletipButton,
+  ToggletipContent,
+} from '../Toggletip';
+import { Information } from '@carbon/icons-react';
+
+export default {
+  title: 'Experimental/unstable__FluidNumberInput',
+  component: FluidNumberInput,
+};
```

Comment:
Could `FluidNumberInputSkeleton` be added as a subcomponent so it shows up in the prop api table on the mdx docs?